### PR TITLE
Add scripts and docs for minimal Metal3 testing

### PR DESCRIPTION
**Pull request by hardys**
Adds some scripts (which currently only work on Linux) to enable minimal testing of Metal<sup>3</sup> on a test cluster deployed via the slemicro scripts.

Until we have aarch64 images for the metal3 components, and the UTM redfish emulation conversation started via #50 is progressed, we'll have to use a Linux host for this setup, but it significantly simplifies the configuration for dev/test iterations compared with the full metal3-demo setup.

### CODE REVIEW

**Review by e-minguez — APPROVED** [review]
I've added some suggestions

File: baremetal_vm/README.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+Then install the chart from the same directory, using the values file:
+
+```
+helm install metal3 . -f ./disable_provnet.yaml 
+```
+
+After some time this should result in a running Metal<sup>3</sup> deployment:
+
+```
+> kubectl get pods
```
[the remark this answers is not in the document]
Comment (hardys) [reply baremetal_vm/README.md:91]: Actually to keep things as simple as possible I've been testing in the default namespace, which works fine - folks can obviously change or add namespaces if they wish though - do you think we need that in the minimal readme instructions?

File: baremetal_vm/README.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+$ curl -s http://192.168.122.1:8000/redfish/v1/Systems/c42def9e-9f16-4aa3-bf0c-d97df0f80a85 | jq .Name
+"BaremetalHost"
+```
+
+Now you can apply the yaml file to provision the Metal<sup>3</sup> BareMetalHost:
+
+```
+$ kubectl apply -f /var/lib/libvirt/images/BaremetalHost.yaml
+
+$ kubectl get bmh
```
[the remark this answers is not in the document]
Comment (hardys) [reply baremetal_vm/README.md:141]: As above, to make things as simple as possible I just used the default namespace for testing

File: baremetal_vm/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (e-minguez) [reply baremetal_vm/README.md:91]: I wasn't sure if the assets were deployed to that namespace and due to the fact you are using the default one, let's ignore my comment then :)

File: baremetal_vm/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (e-minguez) [reply baremetal_vm/README.md:141]: Same, ignore my comment then :)

File: baremetal_vm/README.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+The metal3 chart can be installed with a minimal configuration via the following steps:
+
+Note this assumes that kubectl is configured e.g via the `KUBECONFIG` variable to connect to the management cluster created in the previous step.
+
+First clone the charts repo and create a values file which disables the provisioning network:
+
+```
+git clone https://github.com/suse-edge/charts.git
+cd charts/charts/metal3-deploy/0.1.0
+
```
[the remark this answers is not in the document]
Comment (hardys) [reply baremetal_vm/README.md]: Done

**Review by e-minguez — APPROVED** [review]

**Pull request merged**